linux_os/guide/system/accounts/accounts-session/accounts_users_netrc_file_permissions/oval/shared.xml

```diff
@@ -29,21 +29,36 @@
     <unix:path var_ref="var_accounts_users_home_netrc_file_permissions_dirs"
                var_check="at least one"/>
     <unix:filename operation="pattern match">\.netrc</unix:filename>
-    <filter action="exclude">state_accounts_users_home_netrc_file_permissions</filter>
   </unix:file_object>
 
-  <unix:file_state id="state_accounts_users_home_netrc_file_permissions" version="1">
-    <unix:gread datatype="boolean">false</unix:gread>
-    <unix:gwrite datatype="boolean">false</unix:gwrite>
-    <unix:gexec datatype="boolean">false</unix:gexec>
-    <unix:oread datatype="boolean">false</unix:oread>
-    <unix:owrite datatype="boolean">false</unix:owrite>
-    <unix:oexec datatype="boolean">false</unix:oexec>
+  <unix:file_state id="state_accounts_users_home_netrc_file_permissions_gread" version="1">
```

Comment:
All these 6 states could be unified in a single state, like it was before. The point from my last comment was about moving the `state_accounts_users_home_netrc_file_permissions` as a filter from the `object_accounts_users_home_netrc_file_permissions` object to the `test_accounts_users_home_netrc_file_permissions` object as a state, like you did and is already be fine.

Reply:
I tried to unite them but did not work well since my idea for the implementation was if any of the fields is true we are screwed if I invert them all like a check that all fields are true had problems to implement the inverted "check" for the "file_test"

Reply:
Thanks for the information. It makes sense. I don't think this have any relevant performance impact, so I am fine in any way.

Indeed the logs are easier to interpret. I also liked it. Thanks.